### ISSUE TITLE
OpenGL shaders don't disassemble on new AMD drivers

### ISSUE THREAD

**Issue by baldurk** [GPUOpen-Tools/RGA#37]
On recent drivers (at least 18.12.3 and 19.1.1) running the following command:

`rga -s opengl --isa isa.txt -b out.bin --vert test.vert -c baffin`

for this shader `test.vert`:

```
void main(void) {
  const vec4 verts[4] = vec4[4](vec4(-1.0, -1.0, 0.5, 1.0), vec4(1.0, -1.0, 0.5, 1.0),
                                vec4(-1.0, 1.0, 0.5, 1.0), vec4(1.0, 1.0, 0.5, 1.0));
  gl_Position = verts[gl_VertexID];
}
```

Fails and outputs:

```
Building for Baffin... failed.
Error: failed to generate one or more output files
```

With only the `.bin` file and no ISA file.

Rolling back to driver 18.8.2 works correctly.

Tested on Windows 10 x64 with RGA 2.0.1.

**Comment by AmitBM** [comment]
Hi baldurk,

Thanks for reporting this issue. Work on a fix is in progress.

**Comment by AmitBM** [comment]
Hi baldurk, 

This should be resolved with the latest RGA 2.1 release from today.
